src/index.ts

```diff
@@ -95,13 +95,29 @@ module.exports = class ServerlessPlugin {
 
   private async afterPackageFunction() {
     const config = getConfig(this.serverless.service);
+    const forwarderArn: string | undefined = config.forwarderArn;
+    const forwarder: string | undefined = config.forwarder;
+
+    let datadogForwarderArn;
     if (config.enabled === false) return;
-    if (config.forwarder) {
+    if (forwarderArn && forwarder) {
+      throw new Error(
+        "Both 'forwarderArn' and 'forwarder' parameters are set. Please only use the 'forwarderArn' parameter.",
+      );
+    } else if (forwarderArn !== undefined && forwarder === undefined) {
+      datadogForwarderArn = forwarderArn;
+    } else if (forwarder !== undefined && forwarderArn === undefined) {
+      datadogForwarderArn = forwarder;
+    }
+
+    if (datadogForwarderArn) {
       const aws = this.serverless.getProvider("aws");
-      const errors = await addCloudWatchForwarderSubscriptions(this.serverless.service, aws, config.forwarder);
+      const errors = await addCloudWatchForwarderSubscriptions(this.serverless.service, aws, datadogForwarderArn);
       for (const error of errors) {
         this.serverless.cli.log(error);
       }
+    } else {
+      this.serverless.cli.log("Warning: 'forwarderArn' is not set, skipping CloudWatch Forwarder Subscriptions.");
```

Comment:
I don't think we need a warning here. It's normal to not provide the forwarderArn if you don't want to. There are other ways to set up the subscriptions besides the plugin, so it's not totally unexpected or something we would need to warn customers about.